vendor/github.com/xtaci/smux/session.go

```diff
@@ -3,36 +3,31 @@ package smux
 import (
 	"encoding/binary"
 	"io"
+	"net"
 	"sync"
 	"sync/atomic"
 	"time"
 
 	"github.com/pkg/errors"
+	siasync "github.com/NebulousLabs/Sia/sync" // TODO: Replace with github.com/NebulousLabs/trymutex
```

Comment:
We should move `TryMutex` to it's own top-level package, I think it's both useful and stable.